execution_workflows/Roar/workflow/scripts/Roar.R

```diff
@@ -0,0 +1,109 @@
+
+### Libraries
+
+library(roar)
+library(rtracklayer)
+library(org.Mm.eg.db)
+
+# Input files
+args = commandArgs(trailingOnly=TRUE)
+
+# Input gtf
+gtf <- args[1]
+
+# Input bams
+bam1 <- paste(path=args[2],list.files(path=args[2],pattern="*.bam"),sep="/")
+bam2 <- paste(path=args[3],list.files(path=args[3],pattern="*.bam"),sep="/")
+bams <- c(bam1, bam2)
+# Output prefix
+out.prefix <- args[4]
```

Comment:
So BAM files are collected separately for each condition, then combined into a single vector? I think they should be kept separate as files for each condition are separate arguments for the Roar input function

I'd prefer to have an 'argparse' style input/CLI though, where files are passed for each condition as a separate option (e.g. --condition1 a.bam b.bam --condition2 c.bam d.bam)